ert_gui/tools/plot/plot_api.py

```diff
@@ -1,115 +1,189 @@
+
+    def _get_experiment(self) -> dict:
+        with Storage.session() as client:
+            try:
+                response: requests.Response = client.get("/experiments")
+                response_json = response.json()
+                return response_json[0]
+            except (httpx.RequestError, IndexError) as exc:
+                logger.exception(exc)
+                raise exc
+
+    def _get_ensembles(self, experiement_id) -> List:
+        with Storage.session() as client:
+            try:
+                response: requests.Response = client.get(
+                    f"/experiments/{experiement_id}/ensembles"
+                )
+                response_json = response.json()
+                return response_json
+            except httpx.RequestError as exc:
+                logger.exception(exc)
+                raise exc
 
     def all_data_type_keys(self):
-        """Returns a list of all the keys except observation keys. For each key a dict is returned with info about
+        """Returns a list of all the keys except observation keys.
+
+        The keys are only read from the first ensemble as they should be the same
+
+        For each key a dict is returned with info about
         the key"""
 
-        all_keys = self._facade.all_data_type_keys()
-        log_keys = [k for k in all_keys if k.startswith("LOG10_")]
-
-        return [
-            {
-                "key": key,
-                "index_type": self._key_index_type(key),
-                "observations": self._facade.observation_keys(key),
-                "has_refcase": self._facade.has_refcase(key),
-                "dimensionality": self._dimensionality_of_key(key),
-                "metadata": self._metadata(key),
-                "log_scale": key in log_keys,
-            }
-            for key in all_keys
-        ]
-
-    def _metadata(self, key):
-        meta = {}
-        if self._facade.is_summary_key(key):
-            meta["data_origin"] = "Summary"
-        elif self._facade.is_gen_data_key(key):
-            meta["data_origin"] = "GEN_DATA"
-        elif self._facade.is_gen_kw_key(key):
-            meta["data_origin"] = "GEN_KW"
-        return meta
+        all_keys = []
+        experiment = self._get_experiment()
+        try:
+            ensemble = self._get_ensembles(experiment["id"])[0]
+        except IndexError as exc:
+            logger.exception(exc)
+            raise exec
+
+        with Storage.session() as client:
+            try:
+                response: requests.Response = client.get(
+                    f"/ensembles/{ensemble['id']}/responses", timeout=None
+                )
+                for key, value in response.json().items():
+                    user_data = value["userdata"]
+                    all_keys.append(
+                        {
+                            "key": key,
+                            "index_type": "VALUE",
+                            "observations": self._facade.observation_keys(key),
```

Comment:
This can be changed to "has_observations", which is data we have in the response already. The gui doesn't use the values in the list for anything, only to check whether there is any observations.